tests/test_sqlalchey_seed.py

```diff
@@ -105,13 +127,17 @@ def test_load_fixture(self):
     def test_load_fixtures(self):
         create_table(Base)
         fixtures = load_fixture_files(
-            self.path, ['accounts.yaml', 'pictures.yaml'],
+            self.path, ['accounts.yaml', 'pictures.yaml', 'picture_categories.yaml', 'picture_category_picture.yaml'],
```

Comment:
Please add new test case, such as `test_load_table_fixtures`.
I want `test_load_fixtures` assure not BCBreak.